pontoon/actionlog/admin.py

```diff
@@ -18,6 +19,14 @@ class ActionLogAdmin(admin.ModelAdmin):
         'entity_id',
         'locale_id',
     )
+    fieldsets = (
+        (None, {
+            'fields': (
+                'action_type',
+                'performed_by',
+            ),
+        }),
+    )
```

Comment:
Instead of this, I'd use:
```
raw_id_fields = ('entity', 'translation',)
```

Reply:
Sweet! I went looking for something like this in the docs but failed. Thank you! :slightly_smiling_face: